### PR TITLE
building.md: Add hint to adjust CONFIG_LOCALVERSION (#1407).

### PR DESCRIPTION
**Pull request by olifre**
Adjusting LOCALVERSION ensures the custom kernel
can be identified from uname and distro modules
are not overwritten.

### CODE REVIEW

**Comment by olifre** [comment]
@JamesH65 Thanks from my side, this extra headline clarifies it better that there's an extra paragraph :+1: .

**Comment by lurch** [comment]
@pelwell Should we be recommending editing `.config` directly, rather than using `menuconfig`?  (Is this common practice? :man_shrugging: )

**Comment by olifre** [comment]
@lurch My reasoning for describing how to edit the file was that using `menuconfig` requires installing `libncurses5-dev` which is described in `configuring.md`, so people only reading through `building.md` will not have it installed.

**Comment by lurch** [comment]
So (if I've understood things correctly), I guess if you use both LOCALVERSION (which keeps the modules in a separate directory) and you _also_ use the suggestion from the bottom of this page about using a different filename for the kernel, then reverting to the "stock" kernel would simply be a matter of removing "kernel=kernel-myconfig.img" from `config.txt`? :slightly_smiling_face:

**Comment by olifre** [comment]
You'll still be overwriting the device tree overlays, but that probably does not harm if the version you built is not too distant from the "stock" kernel. Leaving to others if that's a good idea ;-).

**Comment by lurch** [comment]
@JamesH65 This is OK with me, if @pelwell is happy :slightly_smiling_face:

File: linux/kernel/building.md [hunk 1]
```diff
@@ -64,7 +64,9 @@ KERNEL=kernel7l
 make bcm2711_defconfig
 ```
 
-After this is done, you will likely want to adjust the `LOCALVERSION` to ensure your new kernel does not get the same version string as the upstream kernel. This both clarifies you are running your own kernel in the output of `uname` and ensures existing modules in `/lib/modules` are not overwritten.
+#### Customising the Kernel version using LOCALVERSION
```
Comment (pelwell) [inline linux/kernel/building.md]: This would make more sense with one fewer `#`, since it doesn't belong to the list formed by the items above.

Comment (lurch) [reply linux/kernel/building.md]: But OTOH it probably *does* belong under the `### Kernel configuration` section? :man_shrugging: 
Maybe we need a new `#### Apply the default configuration` header, and then move the "Raspberry Pi 4" etc. headers another level down? How many levels of subheadings does https://www.raspberrypi.org/documentation/ support?

**Comment by popcornmix** [comment]
No objections to this change.

File: linux/kernel/building.md (hunk 1 above)
Comment (pelwell) [reply linux/kernel/building.md]: (Editing comments doesn't trigger notifications...)
That looks OK to me.

**Comment by JamesH65** [comment]
Thanks, sorry about the delay.

**Pull request merged**